package.json

```diff
@@ -32,6 +32,7 @@
     "@types/express": "^4.16.0",
     "@types/mocha": "^8.0.3",
     "@types/node": "^10.17.40",
+    "@types/twit": "^2.2.28",
```

Comment:
I reckon this should move to dependencies ya? Otherwise when I deploy this to heroku, it complains. I had to move the other ones as well, except for mocha.

Reply:
This hsould work fine. I startet this on my local machine and had no problem. I have never worked with heroku before, so I don't know if anything is different